src/components/TicketStatusFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import { Checkbox, FormControl, InputLabel, MenuItem, Select, SelectChangeEvent, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { TicketSearchParam } from '../api/searchApi';
+import { TicketStatus } from '../types/publication_types/ticket.types';
+import { dataTestId } from '../utils/dataTestIds';
+
+const ticketStatusValues: TicketStatus[] = ['New', 'Pending', 'Closed', 'Completed'];
+
+export const TicketStatusFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const selectedStatuses = searchParams.get(TicketSearchParam.Status)?.split(',') || ticketStatusValues;
+
+  const handleChange = (event: SelectChangeEvent<string[]>) => {
+    const newSelectedStatuses = event.target.value as TicketStatus[];
+
+    if (newSelectedStatuses.length > 0) {
+      searchParams.set(TicketSearchParam.Status, newSelectedStatuses.join(','));
+    } else {
+      searchParams.delete(TicketSearchParam.Status);
+    }
+
+    history.push({ search: searchParams.toString() });
+  };
+
+  return (
+    <FormControl variant="outlined" size="small" sx={{ minWidth: '15rem' }}>
+      <InputLabel>{t('tasks.status')}</InputLabel>
+      <Select
```

Comment:
Får a11y error på feltet her nå, siden label ikke er koblet til select. Om du vil sette dette selv må vi sette en label-id som samsvarer mellom `<Select>` og `<InputLabel>`, eller så er det mulig å få noe av det samme ved å bruke `<TextField select>` i stedet
```suggestion
      <InputLabel id={selectLabelId}>{t('tasks.status')}</InputLabel>
      <Select
        labelId={selectLabelId}
```